test/unit/metrics/test_metricframe_process_features.py

```diff
@@ -176,3 +176,13 @@ def test_dictionary(self):
         target = _get_raw_MetricFrame()
         result = target._process_features("Unused", rf, y_true)
         self._common_validations(result, ["Alpha", "Beta"])
+
+    def test_tuple_entries(self):
```

Comment:
Could you add a `test_float_entries` to hit that side of the branch. Otherwise, this looks great, thanks!

Reply:
Added test_float_entries to ensure that float values in sensitive_features raise a ValueError.
Also included fixes from pre-commit hooks for whitespace, end-of-file, and import sorting.
Ready for re-review @riedgar-ms 
Thanks